Formula/docbook.rb

```diff
@@ -76,20 +76,29 @@ def install
     end
   end
 
+  def xmlcatalog
+    on_macos do
+      return "xmlcatalog"
+    end
+    on_linux do
+      return "#{Formula["libxml2"].opt_bin}/xmlcatalog"
```

Comment:
How about simply adding `Formula["libxml2"].opt_bin` to `PATH` on linux? Then the rest can be unchanged